test/updater.test.ts

```diff
@@ -0,0 +1,342 @@
+import { mock } from 'jest-mock-extended';
+import {
+  BlockInfo,
+  RuneBlockIndex,
+  RuneEtching,
+  RuneLocation,
+  RuneUtxoBalance,
+  RunestoneStorage,
+} from '../src/indexer/types';
+import { RuneUpdater, UpdaterTx } from '../src/indexer/updater';
+import { BitcoinRpcClient } from '../src/rpcclient';
+import { Network } from '../src/network';
+import { Runestone } from '../src/runestone';
+import { RuneId } from '../src/runeid';
+import { Etching } from '../src/etching';
+import { u128, u32, u64, u8 } from '../src/integer';
+import { Some, None } from '../src/monads';
+import { Rune } from '../src/rune';
+
+function getDefaultRuneUpdaterContext() {
+  const block = { hash: 'hash', height: 100_000, previousblockhash: 'previousblockhash' };
+  const storage = new MemoryStorage();
+  const rpc = mock<BitcoinRpcClient>();
+  const runeUpdater = new RuneUpdater(Network.MAINNET, block, storage, rpc);
+  return { runeUpdater, block, storage, rpc };
+}
+
+class MemoryStorage implements RunestoneStorage {
```

Comment:
Let's move these test utilities to another file

Reply:
Good feedback, will do so in following commit